content/en/continuous-deployment/release-notes/rn-armory-spinnaker/armoryspinnaker_v2-32-0.md

```diff
@@ -91,6 +91,15 @@ Related OSS PRs:
 
 ## Known issues
 <!-- Copy/paste known issues from the previous version if they're not fixed. Add new ones from OSS and Armory. If there aren't any issues, state that so readers don't think we forgot to fill out this section. -->
+### Terraform pipelines S3 Artifact Store does not support AuthZ applications
```

Comment:
## When using S3 artifact stores and Application roles, Terraformer will fail to fetch artifacts

Reply:
Probably could use some other tweaking, as it is right now... it doesn't make a lot of sense

Reply:
Yeah it doesnt. How about now?

Reply:
With S3 artifact storage enabled Terraformer stages break when authentication is set on applications

Reply:
done